### PR TITLE
Display truncated fsf yaml files

### LINKED ISSUE #2267: Snapcraft yaml example in expandable grid pattern should be truncated
### Expected behaviour

Large snapcraft yaml content gets truncated

### Steps to reproduce the problem

Visit https://snapcraft.io/#c

Note that the snapcraft yaml is massive.

image

### PR DESCRIPTION
**Pull request by solazio**
## Done

- Display truncated yaml files on FSF

## Issue / Card

Fixes #2267 

## QA

- Pull the branch
- Run the site using the command `./run`
- View the site locally in your web browser at: http://0.0.0.0:8004/first-snap
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Click on any language and see the yaml files are truncated and you can `Show more` by clicking on the link
- Choosing a different language should reset the previously expanded yaml file to being truncated

### CODE REVIEW

**Comment by Lukewh** [comment]
- Click on "Show more" on the homepage version reloads the page - missing JS?
- Can we not have the code-block match the height of the left-hand content? Some of them seem unbalanced:
Screenshot_2020-02-03 Snapcraft - Snaps are universal Linux packages
- Why has the visual style changed from the graphs on the snap details pages?
- Unrelated, but why has the show-more functionality disappeared from the details pages?

**Comment by danielmutis** [comment]
Link should be black in line with the vanilla "Back to top" link style.

File: static/js/public/fsf-expandable-yaml-details.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+export default function initExpandableYaml() {
+  const showMoreContainer = document.querySelectorAll(
+    "[data-js='js-show-more']"
+  );
+
+  if (showMoreContainer) {
+    showMoreContainer.forEach(el => {
+      const fadeEL = el.querySelector(".p-show-more__fade");
+      const linkEl = el.querySelector(".p-show-more__fade");
```
Comment (Lukewh) [inline static/js/public/fsf-expandable-yaml-details.js]: Should this be the same selector? Maybe `.p-show-more__link`?

Comment (solazio) [reply static/js/public/fsf-expandable-yaml-details.js]: well spotted!

**Comment by barrymcgee** [comment]
# [Codecov](https://codecov.io/gh/canonical-web-and-design/snapcraft.io/pull/2490?src=pr&el=h1) Report
> Merging [#2490](https://codecov.io/gh/canonical-web-and-design/snapcraft.io/pull/2490?src=pr&el=desc) into [master](https://codecov.io/gh/canonical-web-and-design/snapcraft.io/commit/275f4d481828e4e048b5b79898b051f43e0f0a98&el=desc) will **not change** coverage by `%`.
> The diff coverage is `n/a`.

**Pull request merged**